src/i18n/de-de/index.js

```diff
@@ -0,0 +1,358 @@
+            unstake_stlos_success: 'Transaktion erfolgreich! Jetzt prüfen:',
+            deposit_failed: 'Fehler bei der TLOS Einzahlung: { message }',
+            redeem_failed: 'STLOS zu TLOS Konvertierung nicht möglich: { message }',
+            fetch_balance_error: 'Konten-Saldo konnte nicht abgerufen werden: { message }',
+            fetch_stlos_balance_error: 'sTLOS-Saldo konnte nicht abgerufen werden: { message }',
+            fetch_stlos_value_error: 'Wert des sTLOS-Saldo konnte nicht abgerufen werden: { message }',
+            fetch_unstaked_balance_error: 'Abruf des abgehobenen TLOS-Saldos fehlgeschlagen: { message }',
+            fetch_unlocked_balance_error: 'Abruf des einforderbaren sTLOS-Saldos fehlgeschlagen: { message }',
+            fetch_escrow_deposits_error: 'Abruf der ESCROW-Salden fehlgeschlagen: { message }',
+            fetch_conversion_rate_error: 'Abruf des TLOS->sTLOS Umrechnungskurses fehlgeschlagen: { message }',
+            fetch_stlos_contract_error: 'Abruf des sTLOS Smart Contract fehlgeschlagen: { message }',
+            fetch_escrow_contract_error: 'Abruf des ESCROW Smart Contract fehlgeschlagen: { message }',
+            fetch_unstake_period_error: 'Abruf der Sperrfrist fehlgeschlagen: { message }',
+            fetch_account_error: 'Konto konnte nicht abgerufen werden: { message }',
+            fetch_stlos_tvl_error: 'Abruf des sTLOS TVL fehlgeschlagen: { message }',
+            fetch_stlos_apy_error: 'Abruf der sTLOS APY fehlgeschlagen: { message }',
+            fetch_max_deposits_error: 'Abruf der maximalen Abhebungen aus ESCROW Smar Contract fehlgeschlagen: { message }',
+            convert_tlos_to_stlos_error: 'TLOS konnte nicht in STLOS umgewandelt werden: { message }',
+            convert_stlos_to_tlos_error: 'sTLOS konnte nicht in TLOS umgewandelt werden:S { message }',
+            unstake_stlos_error: 'sTLOS konnte nicht abgehoben werden: { message }',
+            withdraw_failed: 'TLOS Position konnte nicht ausgezahlt werden: { message }',
+        },
+        explore_transactions: 'Transaktionen prüfen',
+        recent_transactions: 'Neueste Transaktionen',
+        telos_evm_explorer: 'Telos EVM Explorer',
+        rpc_endpoints: 'RPC Endpunkte',
+        monitor: 'Monitor',
+        oops: 'Oh. Hier existiert nichts...',
+        go_home: 'Home',
+        select_sol_file: 'Bitte .sol Smart Contract Datei für Hochladen auswählen',
+        select_json_file: 'Bitte standard JSON Objekt-Datei für Hochladen auswählen',
+        paste_contract_contents: 'Sie müssen eine Datei zum Hochladen auswählen oder den Reiter wechseln, um Smart Contract Inhalte direkt einzufügen.',
+        contract_address: 'Smart Contract Adresse',
+        enter_contract_address: 'Bitte Smart Contract Adresse eingeben \'0x0123...\'',
+        invalid_address_format: 'Ungültiges Adress-Format',
+        compiler_version: 'Compiler Version',
```

Comment:
This line needs translation

Reply:
@Viterbo This is Ok "Compiler Version" does not require translation as Version = version ;-) Compiler does not need translation as used by programers in German too